src/20231205_AaveV3Ethereum_IncreaseGHOBorrowRate100BpsTo635OnAaveV3/IncreaseGHOBorrowRate100BpsTo635OnAaveV3.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Increase GHO Borrow Rate 100 bps to 6.35% on Aave V3"
+author: "Alice Rozengarden (@Rozengarden - Aave-chan initiative)"
+discussions: "https://governance.aave.com/t/arfc-increase-gho-borrow-rate-100-bps-to-6-35-on-aave-v3/15744"
+---
+
+## Simple Summary
+
+This AIP proposes an increase in the GHO borrow rate from 5.35% APY to 6.35% APY to support the GHO peg restoration and align borrowing costs with market rates.
```

Comment:
the change is from 5.22 to 6.22 apr - didn't do the math but would assume the same message with `apy` is wrong due to compounding.

Reply:
you're correct, we updated the math. thanks for the feedback.